P/Pikchr/bundled/README

```diff
@@ -0,0 +1,5 @@
+The Pikchr library homepage is available at https://pikchr.org
+
+The code is hosted on Fossil SCM, and is originally written as pikchr.y. As the ".y" suffix implies, this file is a grammar specification intended as input to the yacc-like LALR(1) parser generator program "Lemon".
+
+As a convinience, the result of the parser generator is checked into the Pikchr source repo as pikchr.c. We pick up this file directly, and use it to build the library. This is useful since running target executables during build is problematic in BinaryBuilder. 
```

Comment:
We can run native executables (something built for `${MACHTYPE}`, aka `x86_64-linx-musl`), if that helps.

Reply:
Yeah, but then I have to compile half the sources with native compiler, the other half with the target compiler. Not worth the complication since the nice people at Pikchr check in the generated code.

Reply:
Ok, cool!

Reply:
Also, this is using a three line cmake recipe, which is pretty much the upper limit of my cmake skills.